shoes-core/lib/shoes/star.rb

```diff
@@ -39,28 +39,24 @@ def in_bounds?(x, y)
 
     def redraw_left
       return 0 unless element_left
-      if center
-        element_left - width * 0.5 - style[:strokewidth].to_i
-      else
-        super
-      end
+      calculated_left = super
+      calculated_left = element_left - width * 0.5 if center
+      calculated_left - style[:strokewidth].to_i * 2
```

Comment:
don't we have a `strokewidth` method or something defined that already returns the integer?

Reply:
We do indeed have a method and [a style default to 1](https://github.com/shoes/shoes4/blob/master/shoes-core/lib/shoes/common/style.rb#L15), so switching this over. Great catch @PragTob!